site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
```

Comment:
These dates are not guaranteed.  I wonder if we should include some general language to that effect somewhere?

Reply:
```suggestion
### Chrome 91 ([targeting May 25, 2021][release-schedule])

[release-schedule]: https://www.chromestatus.com/features/schedule
```

Attempts to address Devlin's concerns by using future-facing language. Link optional.